framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -213,9 +213,10 @@ struct BPL_WLAN_IFACE {
  *
  * @return 1 if enabled.
  * @return 0 if disabled.
+ * @return default_value if value not set
  * @return -1 Error.
  */
-int cfg_is_enabled();
+int cfg_is_enabled(int default_value = -1);
```

Comment:
I don't think default value should be a parameter - for each case we should handle on its own and decide if a default value should be used or if that specific parameter is mandatory.
In the `cfg_is_enabled` case - this should be mandatory.